File: Plugins/Aspose_Cells_Java_for_PHP/src/aspose/cells/WorkingWithFiles/UtilityFeatures/ConvertingExcelFilesToHtml.php
<?php


namespace Aspose\Cells\WorkingWithFiles\UtilityFeatures;

use com\aspose\cells\Workbook as Workbook;
use com\aspose\cells\SaveFormat as SaveFormat;


class ConvertingExcelFilesToHtml {

    public static function run($dataDir=null)
    {
        $saveFormat = new SaveFormat();

        $workbook = new Workbook($dataDir . "Book1.xls");

        //Save the document in PDF format
        $workbook->save($dataDir . "OutBook1.html", $saveFormat->HTML);

        // Print message
        echo "\n Excel to HTML conversion performed successfully.";
    }

}

File: Plugins/Aspose-Cells-Java-for-Jython/asposecells/WorkingWithFiles/ConvertingExcelFilesToHtml.py
from asposecells import Settings
from com.aspose.cells import Workbook
from com.aspose.cells import SaveFormat


class ConvertingExcelFilesToHtml:

    def __init__(self):
        dataDir = Settings.dataDir + 'WorkingWithFiles/ConvertingExcelFilesToHtml/'
        
        saveFormat = SaveFormat

        workbook = Workbook(dataDir + "Book1.xls")

        #Save the document in PDF format
        workbook.save(dataDir + "OutBook1.html", saveFormat.HTML)

        # Print message
        print "\n Excel to HTML conversion performed successfully."
 
if __name__ == '__main__':        
    ConvertingExcelFilesToHtml()